lib/web_ui/dev/driver_manager.dart

```diff
@@ -50,17 +77,19 @@ class ChromeDriverManager extends DriverManager {
   /// Driver should already exist on LUCI as a CIPD package.
   @override
   Future<void> _verifyDriverForLUCI() {
-    if (!_browserDriverDir.existsSync()) {
+    if (!_browserDriverDirWithVersion.existsSync()) {
       throw StateError('Failed to locate Chrome driver on LUCI on path:'
-          '${_browserDriverDir.path}');
+          '${_browserDriverDirWithVersion.path}');
     }
     return Future<void>.value();
   }
 
   @override
-  Future<void> _startDriver(String driverPath) async {
+  Future<void> _startDriver() async {
     await startProcess('./chromedriver/chromedriver', ['--port=4444'],
-        workingDirectory: driverPath);
+        workingDirectory: isLuci
+            ? _browserDriverDirWithVersion.path
+            : _browserDriverDir.path);
```

Comment:
Can we remove this distinction? Why not use `_browserDriverDirWithVersion` on both LUCI and local?

Reply:
done. thanks, code is a lot simpler now. :+1: :sunglasses: